.eslintrc.js

```diff
@@ -1,12 +1,31 @@
 module.exports = {
+	parser: '@babel/eslint-parser',
 	extends: [ 'plugin:@woocommerce/eslint-plugin/recommended' ],
 	globals: {
 		_: false,
 		Backbone: false,
 		jQuery: false,
 		wp: false,
 	},
+	env: {
+		browser: true,
+		'jest/globals': true,
+		node: true,
+	},
+	rules: {
+		'@woocommerce/dependency-group': 'off',
```

Comment:
This doesn't play nice with the webpack alias ( https://github.com/WordPress/gutenberg/issues/16789 )

Reply:
I've been "solving" this issue by selectively [turning off the rule only for the specific imports this affects](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1825/files#diff-14e32414b0840eecfb83ae51a4ab3ff47cb4f1146ed445ff109ad7941dc3d0aeR10-R16), but I have wondered if simply turning off the rule is better. I'm not sure which I'd prefer tbh 🤔 

Also, I had to add a specific webpack resolver for ESLint so it'd be able to actually resolve the path aliases in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1825/commits/2af1b3c3ffae63e9d09d5bf62300b2c76a022186. That commit should probably be part of this PR instead.

Reply:
And that commit is (of course) missing the package.json update because I included that in the [following commit](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1825/commits/475253dad0cd8afa32a21a541705659e76ce824a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R50) 🙃

Reply:
Oh yeah, I also [had to add the resolver](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/6155feeeb4ddb3211a0b3c557ad7dd6a0f2db0a3/.eslintrc.js#L28) to the eslint config 👍 

In Calypso, they also ditched their equivalent of the `@woocommerce/dependency-group` rule in favor of `import/order` - I wonder if we should do the same 🤷 : p4TIVU-9Hd-p2

In the meantime, would you rather like me to remove this line and add eslint disable rules where necessary?

Reply:
Personally I'm in favor of adopting `import/order`; I really don't like the Internal/External doc comment split 😅 

Let's bring this up in Slack (#woo-stripe) with a simple yes/no vote via emoji reactions 🤔 
@frosso would you mind taking care of that?

Reply:
I split the PR in 2, to make a bit easier to review (I hope) - 06eadaf2b1dba7629cd17a5c2d67f3f10a2eb185 implements the rules and b475885880a2ed29f2037e9cd1d48b9b3e081a7b fixes them.

We're now using `import/order`.

Reply:
I'm going to try and keep this PR updated, as new changes come in. So those hashes might change in the future if I decide to rebase.

Reply:
Hopefully you won't have to keep it up to date much longer; merging tomorrow would be amazing 🥇